diff --git a/Makefile b/Makefile
index 9e405218162b..21c840838976 100644
--- a/Makefile
+++ b/Makefile
@@ -1713,7 +1713,7 @@ CLEAN_FILES += vmlinux.symvers modules-only.symvers \
 # Directories & files removed with 'make mrproper'
 MRPROPER_FILES += include/config include/generated          \
                  arch/$(SRCARCH)/include/generated .objdiff \
-                 debian snap tar-install PKGBUILD pacman \
+                 snap tar-install PKGBUILD pacman \
                  .config .config.old .version \
                  Module.symvers \
                  certs/signing_key.pem \
