--- Makefile.orig	2026-08-12 11:18:26.968139188 +0200
+++ Makefile	2026-08-12 11:18:33.814536000 +0200
@@ -1768,7 +1768,7 @@
 # Directories & files removed with 'make mrproper'
 MRPROPER_FILES += include/config include/generated          \
 		  arch/$(SRCARCH)/include/generated .objdiff \
-		  debian snap tar-install PKGBUILD pacman \
+		  snap tar-install PKGBUILD pacman \
 		  .config .config.old .version \
 		  Module.symvers \
 		  certs/signing_key.pem \
