--- output/build/qt5base-bebdfd54917e25d1c100e6bd9f5dd53c2e645fd8/src/corelib/global/qfloat16_f16c.c	2026-08-14 14:25:21.961764809 +0200
+++ output/build/qt5base-bebdfd54917e25d1c100e6bd9f5dd53c2e645fd8/src/corelib/global/qfloat16_f16c.c	2026-08-14 14:24:20.955822161 +0200
@@ -48,6 +48,7 @@
 extern "C" {
 #endif
 
+#if 0
 QT_FUNCTION_TARGET(F16C)
 void qFloatToFloat16_fast(quint16 *out, const float *in, qsizetype len) Q_DECL_NOEXCEPT
 {
@@ -79,7 +80,7 @@
     for (epilog_i = 0; i < len && epilog_i < 3; ++i, ++epilog_i)
         out[i] = _mm_cvtss_f32(_mm_cvtph_ps(_mm_cvtsi32_si128(in[i])));
 }
-
+#endif
 #ifdef __cplusplus
 } // extern "C"
 QT_END_NAMESPACE
